tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+test_name: hpc-slurm-chromedesktop
+deployment_name: "enter-{{ build }}"
+# Manually adding the slurm_cluster_name for use in node names, which filters
+# non-alphanumeric chars and is capped at 10 chars.
+slurm_cluster_name: "enter{{ build[0:5] }}"
+zone: europe-west1-d
+cli_deployment_vars:
+  region: europe-west1
+  zone: "{{ zone }}"
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/community/examples/hpc-slurm-chromedesktop.yaml"
+network: "{{ deployment_name }}-net"
+max_nodes: 5
+# Note: Pattern matching in gcloud only supports 1 wildcard.
+login_node: "{{ slurm_cluster_name }}-login-*"
+controller_node: "{{ slurm_cluster_name }}-controller"
+post_deploy_tests: []
```

Comment:
Please add partitions and mounts to check

Reply:
Added, mount and partition test. It seems we had set dynamic node in the blueprint as 1 so partition test timed out. So, also updated that.